src/kinetics/RxnRates.cpp

```diff
@@ -51,6 +51,28 @@ void Arrhenius::setParameters(const AnyValue& rate,
     }
 }
 
+BlowersMasel::BlowersMasel()
+    : m_logA(-1.0E300)
+    , m_b(0.0)
+    , m_A(0.0)
+    , m_w(0.0)
+    , m_E0(0.0)
+{
+}
+
+BlowersMasel::BlowersMasel(doublereal A, doublereal b, doublereal E0, doublereal w)
```

Comment:
Replace `doublereal` with `double`. Applies here and in `BMSurfaceArrhenius::addCoverageDependence`